src/actions/action.js

```diff
@@ -29,16 +29,25 @@ const getActions = async (
 ) => async (dispatch, getState) => {
   dispatch(flagGettingActions(true));
   try {
-    const { apiHost, spaceId: currentSpaceId } = getApiContext(getState);
+    const { apiHost, spaceId: currentSpaceId, standalone } = getApiContext(
+      getState,
+    );
+
+    // if standalone, you cannot connect to api
+    if (standalone) {
+      return false;
+    }
 
     // by default include current space id
-    const { spaceId = [] } = params;
-    if (!spaceId.length) {
-      spaceId.push(currentSpaceId);
+    const completeParams = { ...params };
+    if (!completeParams.spaceId || !completeParams.spaceId.length) {
```

Comment:
Can you use `?.` here?

Reply:
I think that should work but will leave as is for now so that we stay consistent with apps 2 and 3